tests/Notification/NotifierTest.php

```diff
@@ -208,7 +207,14 @@ public function testPrepare($language, $user, $changeName, $changeAvatar, $name,
 				->willReturnSelf();
 		}
 
-		$return = $this->notifier->prepare($notification, $language);
-		$this->assertEquals($notification, $return);
+		try {
+			$return = $this->notifier->prepare($notification, $language);
+			$this->assertEquals($notification, $return);
+		} catch (\Exception $exception) {
+			if ($subjectContains === false) {
+				$this->assertInstanceOf(InvalidArgumentException::class, $exception);
+			}
```

Comment:
Shouldn't the else branch throw the exception again?

Reply:
Yes, good catch. Fixed.